osu.Framework/Platform/SDL/SDL3GraphicsSurface.cs

```diff
@@ -216,5 +216,13 @@ bool IOpenGLGraphicsSurface.VerticalSync
         bool ILinuxGraphicsSurface.IsWayland => window.IsWayland;
 
         #endregion
+
+        #region Android-specific implementation
+
+#pragma warning disable CA1416 // This is only usable in Android
+        IntPtr IAndroidGraphicsSurface.JniEnvHandle => SDL3.SDL_AndroidGetJNIEnv();
+#pragma warning restore CA1416
```

Comment:
```suggestion
        [SupportedOSPlatform("android")]
        IntPtr IAndroidGraphicsSurface.JniEnvHandle => SDL3.SDL_AndroidGetJNIEnv();
```

Strangely, this won't cause platform warnings elsewhere.